README.md

```diff
@@ -7,6 +7,8 @@ This plugin requires at least Sublime Text 3 v.3176 or higher for best performan
 ## Installing
 - Move directory "SdlLogs" to Sublime Packages folder.
 (Generally at /home/*USER*/.config/sublime-text-3/Packages)
+- Should install package for yaml
```

Comment:
@dkandyba extract list of dependencies to separate block

Reply:
@LuxoftAKutsan fixed in def3edd1f30bd4e9f9281615b12f602163fb2ac4